waterlog/.env.development

```diff
@@ -1 +1 @@
-REACT_APP_API_URL=https://localhost:5001
\ No newline at end of file
+REACT_APP_API_URL=https://localhost:44383
```

Comment:
dont commit `.env` files

Reply:
We are not storing any passwords, its just url.